### DISCUSSION TITLE
OSError: libcublas.so.11: cannot open shared object file: No such file or directory

### DISCUSSION THREAD

**Discussion by raianand1991** [openai/whisper#663] [Q&A] [answered]
I am getting this error after installing whisper

**Comment by jongwook** [comment, marked as answer]
It appears that PyTorch (with CUDA support) is not properly installed on your system. You could try reinstalling PyTorch by following the [official website](https://pytorch.org/get-started/locally/).

If you have an NVIDIA GPU in your computer, the Conda command will probably work better for you, since it will also install the required CUDA toolkit.

If you don't have a GPU in your system, please select "CPU" in the "Compute Platform" row, so that PyTorch does not attempt to load CUDA libraries.
